packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -268,6 +269,13 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to being adapted to the current [TargetPlatform].
   final bool centerTitle;
 
+  /// The margin between [title] and [leading] content. This margin is applied
```

Comment:
Isn't this the margin around the title on both sides?

Reply:
You are right. I was confused about this due to wording and the tests. Fixed.